src/app/ReadHandler.h

```diff
@@ -212,15 +212,15 @@ class ReadHandler : public Messaging::ExchangeDelegate
 
     enum class ReadHandlerFlags : uint8_t
     {
-        // mHoldReport is used to prevent subscription data delivery while we are
+        // mHoldMinInterval is used to prevent subscription data delivery while we are
```

Comment:
Pre-existing, but these "m" prefixes don't exist now that these are flags, not member vars.